README.md

```diff
@@ -3,13 +3,16 @@
 This is an [Arcanist][] extension that displays file ownership information.
 It is implemented as an `arc owners` command:
 
-    owners [path ...]
+    owners [options] [path ...]
         Supports: git, hg
         Display ownership information for a list of files.
 
         Without paths, the files changed in your local working copy will
         be used.
 
+        --output format
+            With 'json', show owners in machine-readable JSON format.
```

Comment:
Maybe add a "Supports" line, as above:

```
Supports: json, text
```

Also, if there's a convention to say that the default is "text", that would be nice to add.